.github/workflows/check-examples.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         base: ${{ github.ref }}
         filters: |
           examples:
-            - 'examples/**'
+            - '{examples/**, .github/**}'
```

Comment:
Is the following '.github/workflows/check-examples.yml' covered by .github/**?

Reply:
I have cleaned that up in [4def276](https://github.com/CQCL/pytket-quantinuum/pull/268/commits/4def2761cfcb036437741d1168759f9e9dd4b301)